### DISCUSSION TITLE
Gene based subsetting of cells in ArchR object

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1490] [General]
If I have an ArchR object and I'm interested in isolating a subset of cells across several clusters that expresses a given gene how can this be accomplished?

**Comment by rcorces** [comment]
Please edit your post to have a more descriptive title and more details in your question.

**Reply by sylestiel** [reply]
Made changes. Awaiting a reply

**Comment by rcorces** [comment]
Assuming you have GeneExpressionMatrix or GeneIntegrationMatrix, I would use `getMatrixFromProject()` to get the matrix, Figure out which cells express your gene of interest and get their `cellNames`, and then use `subsetArchRProject()` to get those cells.

**Comment by sylestiel** [comment]
Can you elaborate on " Figure out which cells express your gene of interest and get their cellNames, and then use subsetArchRProject() to get those cells"
